src/v2/Apps/Authentication/Utils/useAuthForm.ts

```diff
@@ -54,6 +55,7 @@ export function useAuthForm({ canonical, pageTitle, type }: UseAuthFormProps) {
   const options: ModalOptions = {
     action,
     afterSignUpAction,
+    api_login,
```

Comment:
This is necessary to add here so that if you navigate between `/login` and `/signup` via the links on the form, the query parameter is passed through.

Also-- I'm not sure whether to prefer camelcase or snakecase here. The parameter originates from gravity which is why I chose snake_case, but I'm not tied to it.

Reply:
Let's do camel case -- the loose reason being, after a parameter is passed into a system the new system owns it; it can then get passed around further without knowledge of the originating system and we'd want to stay closest to the idioms within the new owner.

Reply:
+1 to consistent camel case. If querystring parameters are the "API" exposed by Force, Gravity should consume it according to Force's conventions.